src/components/CytoscapeGraph/FocusAnimation.ts

```diff
@@ -59,17 +59,19 @@ export default class FocusAnimation {
       }
       const current = Date.now();
       const step = (current - this.startTimestamp) / ANIMATION_DURATION;
-      this.layer.clear(this.context);
-      this.layer.setTransform(this.context);
 
-      if (step >= 1) {
+      if (step >= 1 || !this.layer) {
         this.stop();
         if (this.onFinishedCallback) {
           this.onFinishedCallback();
         }
+
         return;
       }
 
+      this.clear();
+      this.layer.setTransform(this.context);
```

Comment:
This code is no longer needed.
The error was caused by running this method without properly binding "this".
We probably want to know when this breaks and this will silently fail.

https://github.com/kiali/kiali-ui/pull/1293

Reply:
The whole file? This part of the patch? Or the whole commit?

Reply:
Just this file